spec/controllers/idv/phone_question_controller_spec.rb

```diff
@@ -62,43 +94,85 @@
+          subject.idv_session.flow_path = 'standard'
 
-        get :show
+          get :show
+
+          expect(response).to redirect_to(idv_document_capture_url)
+        end
+
+        it 'redirects to link_sent in hybrid flow' do
+          subject.idv_session.flow_path = 'hybrid'
 
-        expect(response).to redirect_to(idv_document_capture_url)
+          get :show
+
+          expect(response).to redirect_to(idv_link_sent_url)
+        end
       end
 
-      it 'redirects to link_sent in hybrid flow' do
-        subject.idv_session.flow_path = 'hybrid'
+      context 'on mobile device' do
+        it 'redirects to document_capture' do
+          subject.idv_session.skip_hybrid_handoff = true
 
-        get :show
+          get :show
 
-        expect(response).to redirect_to(idv_link_sent_url)
+          expect(response).to redirect_to(idv_document_capture_url)
+        end
+      end
+
+      context 'hybrid flow is not available' do
+        before do
+          allow(FeatureManagement).to receive(:idv_allow_hybrid_flow?).and_return(false)
+        end
+
+        it 'redirects the user straight to document capture' do
+          get :show
+          expect(response).to redirect_to(idv_document_capture_url)
+        end
       end
     end
+  end
 
-    context 'on mobile device' do
-      it 'redirects to hybrid_handoff controller' do
-        subject.idv_session.skip_hybrid_handoff = true
+  describe '#phone_with_camera' do
+    let(:analytics_name) { :idv_doc_auth_phone_question_submitted }
 
-        get :show
+    it 'redirects to hybrid handoff' do
+      get :phone_with_camera
 
-        expect(response).to redirect_to(idv_document_capture_url)
-      end
+      expect(response).to redirect_to(idv_hybrid_handoff_url)
     end
 
-    context 'hybrid flow is not available' do
-      before do
-        allow(FeatureManagement).to receive(:idv_allow_hybrid_flow?).and_return(false)
-      end
+    it 'sends analytics submitted event' do
+      get :phone_with_camera
 
-      it 'redirects the user straight to document capture' do
-        get :show
-        expect(response).to redirect_to(idv_document_capture_url)
-      end
+      expect(@analytics).
+        to have_logged_event(analytics_name, analytics_args.merge!(phone_with_camera: true))
+    end
+  end
+
+  describe '#phone_without_camera' do
+    let(:analytics_name) { :idv_doc_auth_phone_question_submitted }
+
+    it 'redirects to hybrid handoff' do
+      get :phone_without_camera
+
+      expect(response).to redirect_to(idv_hybrid_handoff_url)
+    end
+
+    it 'sends analytics submitted event' do
+      get :phone_without_camera
+
+      expect(@analytics).
+        to have_logged_event(analytics_name, analytics_args.merge!(phone_with_camera: false))
+    end
+
+    it 'set idv_session flow path to standard' do
+      expect do
+        get :phone_without_camera
+      end.to change { subject.idv_session.flow_path }.from(nil).to eql('standard')
```

Comment:
Don't need `eql` since `.to change` builds in comparison. And I think the curly brace form is a little easier to read.
```suggestion
      expect { get :phone_without_camera }.
        to change { subject.idv_session.flow_path }.from(nil).to 'standard'
```